src/dbsync/tests/utils/threadDispatcher_test.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
```

Comment:
Please, use: 
#ifndef _THREAD_DISPATCHER_TEST_H
#define _THREAD_DISPATCHER_TEST_H
...
#endif // _THREAD_DISPATCHER_TEST_H

instead of #pragma once.